### ISSUE TITLE
Python 3.8 ImportError: cannot import name 'clock' from 'time'

### ISSUE THREAD

**Issue by sergiopasra** [PyTables/PyTables#744]
`time.clock` will be removed in Python 3.8 after being deprecated in Python 3.3

```
$ python3.7 -c 'from time import clock; clock()'
-c:1: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
```
The recommendation is to change `clock` by either `time.perf_counter`or `time.process_time`

The differences between both functions are subtle, but it seems that `timeit`is using `time.perf_counter`, so I would use that function

See also: https://www.webucator.com/blog/2015/08/python-clocks-explained/

**Comment by tomkooij** [comment]
Fixed in #745

**Comment by pruthviraja786** [comment]
Thanks for the suggestion for the time module, "from time import perf_counter" works without any error. Try using perf_counter to count the time elapsed to execute the code.
